src/main/java/com/example/timskiproekt/controller/AddressController.java

```diff
@@ -1,14 +1,32 @@
 package com.example.timskiproekt.controller;
 
+import com.example.timskiproekt.domain.Address;
 import com.example.timskiproekt.service.AddressService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 @RequestMapping("/addresses")
 @RequiredArgsConstructor
 public class AddressController {
 
     private final AddressService addressService;
+
+    @GetMapping("/")
```

Comment:
Please remove this line. It is sufficient, since this mapping it's already defined on class level.

Reply:
I think we should keep it since it signifies the end of URL resource path.